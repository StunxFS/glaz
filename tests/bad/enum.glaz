enum BadField {
    Field = true
}

enum DupFields {
	Field1, // 0
	Field2 = 0, // BAD

	Field3 = 1,
	Field4 = 1
}

fn main() {}
