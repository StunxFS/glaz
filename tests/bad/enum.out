tests/bad/enum.glaz:2:14: error: default value for enum has to be an integer
     | 
   2 |     Field = true
     |             ^^^^
tests/bad/enum.glaz:7:3: error: duplicate enum value `0`
     | 
   7 | 	Field2 = 0, // BAD
     | 	^^^^^^
tests/bad/enum.glaz:10:3: error: duplicate enum value `1`
     | 
  10 | 	Field4 = 1
     | 	^^^^^^
glaz error: could not compile `enum`, aborting due to 3 previous errors.
